pkg/k8s/apps/crud.go

```diff
@@ -82,6 +83,21 @@ func HasBeenChanged(k8sObject *model.K8sObject) bool {
 	}
 }
 
+func ValidateMountPaths(k8sObject *model.K8sObject, dev *model.Dev) error {
+	devContainer := GetDevContainer(&k8sObject.PodTemplateSpec.Spec, dev.Container)
+
+	for _, vm := range devContainer.VolumeMounts {
+		for _, syncVolume := range dev.Sync.Folders {
+			if vm.MountPath == syncVolume.RemotePath {
```

Comment:
I think this is only an error if persistent volume is enabled in the okteto manifest.